lib/hm/dig.rb

```diff
@@ -0,0 +1,21 @@
+class Hm
+  module Dig
+    DIGGABLE_CLASSES = [Hash, Array].freeze
```

Comment:
Probably `Struct` and `OpenStruct` can be added in future (at least to provide consistent "deep dig"), please just leave a `# TODO:` comment here about that.